Fail to build on macOS

Version: latest master commit 09fce547dfadc824686aa0bbf4935a04491a0790 

How to reproduce:

```shell
$ mkdir build
$ cd build
$ cmake ..
-- Found Python3Interp: /opt/homebrew/bin/python3.9
-- Found python 3 version 3.9
-- Using PYTHON3_SITEPKG_DIR=/opt/homebrew/opt/python@3.9/Frameworks/Python.framework/Versions/3.9/lib/python3.9/site-packages
-- Found Perl: /usr/bin/perl (found version "5.30.3")
-- Could NOT find PerlLibs (missing: PERL_INCLUDE_PATH) (found version "5.30.3")
CMake Error at libproxy/cmake/modules/pacrunner_duktape.cmk:10 (px_check_modules):
  Unknown CMake command "px_check_modules".
Call Stack (most recent call first):
  libproxy/cmake/modules.cmk:21 (include)
  libproxy/CMakeLists.txt:14 (include)
```

--- a/libproxy/cmake/pkgconfig.cmk
+++ b/libproxy/cmake/pkgconfig.cmk
@@ -1,4 +1,4 @@
-if(NOT WIN32 AND NOT APPLE)
+if(NOT WIN32)
   find_package(PkgConfig)
 
   # Define our magical px_check_modules function

